pom.xml

```diff
@@ -28,13 +28,13 @@
 
     <!-- Properties -->
     <properties>
-        <version.arquillian>1.1.11.Final</version.arquillian>
-        <version.arquillian.cube>1.0.0.Alpha16</version.arquillian.cube>
+        <version.arquillian>1.1.13.Final</version.arquillian>
+        <version.arquillian.cube>1.6.1-SNAPSHOT</version.arquillian.cube>
```

Comment:
I think we could
1. Wait for 1.6.2-Final (we can even ask them to release it ASAP)
2. Change it to 1.6.1-Final
3. Go for it anyway but then we should have our own local copy of it (local, ci, etc)

I'd go with option 1, but this is up to you, really.

Going back to vacations!! :D